lib/cucumber/multiline_argument/data_table.rb

```diff
@@ -75,7 +75,9 @@ def row(row)
         def eof; end
       end
 
-      NULL_CONVERSIONS = Hash.new(strict: false, proc: ->(cell_value) { cell_value }).freeze
+      # This is a Hash being initialized with a default value of a Hash, DO NOT REFORMAT TO REMOVE {}
+      # Future versions [3.4.0+] of ruby will interpret these as keywords and break.
+      NULL_CONVERSIONS = Hash.new({ strict: false, proc: ->(cell_value) { cell_value } }).freeze
```

Comment:
Why not simply:
```ruby
NULL_CONVERSIONS = { strict: false, proc: ->(cell_value) { cell_value } }.freeze
```
? The `Hash.new` seems redundant here.
Unless `Hash` here is not `::Hash`?

Reply:
I'll be honest, this is very old code and I don't know the precise way it is used. This was the smallest safest change suggested.